statler_site/views.py

```diff
@@ -9,7 +9,8 @@ def healthCheck(request):
 def getPlayListPage(request):
     """returns an html page which will use JavaScript
     to get and display data"""
-    return render_to_response('play-list-page.html')
+    # 'all'  is the hardcoded play list name for getting all plays
```

Comment:
Also, returning just the plays creates a new JSON signature coming out of the API, which seems to add unnecessary complexity at the API level.